src/Controllers/BeheerEventAanmakenController.php

```
@@ -18,32 +21,35 @@ public function step2() {
         $hoofdEvent = $_POST['hoofdEvent'] ?? null;
         $eventID = $_POST['eventID'] ?? null;
-        $date = $_POST['datum'] ?? null;
-        $startTime = $_POST['begin-tijd'] ?? null;
-        $endTime = $_POST['eind-tijd'] ?? null;
+
+
+        $dates = $_POST['datum'] ?? [];
+        $startTimes = $_POST['begin-tijd'] ?? [];
+        $endTimes = $_POST['eind-tijd'] ?? [];
+
+        $hasAtLeastOneTime = !empty($dates) && !empty($startTimes) && !empty($endTimes);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$hasAtLeastOneTime'. (undefined)

(UnusedLocalVariable)